partiql-ast/src/main/java/org/partiql/ast/v1/FromJoin.java

```diff
@@ -17,18 +17,18 @@
 @EqualsAndHashCode(callSuper = false)
 public class FromJoin extends FromTableRef {
     @NotNull
-    public final From lhs;
+    public final FromTableRef lhs;
 
     @NotNull
-    public final From rhs;
+    public final FromTableRef rhs;
```

Comment:
This was an error in the original PR adding the hand-written ASTs. The `lhs` and `rhs` should have been `FromTableRef`s.